packages/pwa-kit-react-sdk/src/ssr/universal/components/redirect-with-status/index.test.js

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
You'll not only want to add a unit test for the component, but you'll want to add a test in the `react-rendering.test.js` file. Look at around line 305 for how we are currently resting the redirect behaviour. You'll want to add a similar test that redirects with a custom redirect status code.

Reply:
A new test inside `react-rendering.test.js` has been added!